### PR TITLE
Setup Flake8 linter for CI

### PR DESCRIPTION
**Pull request by TimothyClaeys**

### CODE REVIEW

File: openvisualizer/client/plugins/msf.py [hunk 1]
```diff
@@ -17,7 +15,7 @@ def __init__(self, proxy, mote_id, refresh_rate):
 
     def render(self, ms=None):
         super(MSF, self).render()
-        msf_values = json.loads(ms[MoteState.ST_MSF])
+        # msf_values = json.loads(ms[MoteState.ST_MSF])
```
Comment (changtengfei) [inline openvisualizer/client/plugins/msf.py:18]: Is this intended to be removed?

File: openvisualizer/client/webserver.py [hunk 2]
```diff
@@ -232,7 +232,8 @@ def _topology_motes_update(self):
 
         motes_temp = {}
         for (k, v) in bottle.request.forms.items():
-            m = re.match("motes\[(\w+)\]\[(\w+)\]", k)
+            print k
```
Comment (changtengfei) [inline openvisualizer/client/webserver.py]: should this only for debugging?

**Review by changtengfei — COMMENTED** [review]
Cool! @TimothyClaeys only two comments on this PR. I will merge this PR once it passes the Travis build

File: openvisualizer/client/plugins/msf.py (hunk 1 above)
Comment (TimothyClaeys) [reply openvisualizer/client/plugins/msf.py:18]: Yes for now it is. The MSF viewer is for a future feature that is not yet implemented.

File: openvisualizer/client/webserver.py (hunk 2 above)
Comment (TimothyClaeys) [reply openvisualizer/client/webserver.py]: Yes, forgot to remove, will fix.

**Pull request merged**